bool ppp_start

---

bool ppp_done
